agent.html

```diff
@@ -383,7 +383,7 @@ <h3>Completed</h3>
 		<audio id="agent-remote-audio" autoplay></audio>
 		<!-- //blog -->
     <script src="https://unpkg.com/webex/umd/webex.min.js"></script>
-	<script src='https://unpkg.com/webex@2.60.0-next.2/umd/calling.min.js'></script>
+	<script src='https://unpkg.com/webex@3.5.0-next.25/umd/calling.min.js'></script>

```

Comment:
Please change this to 3.7.0.. that supports guest calling right?

Reply:
Even 3.5.0 supports guest calling